Apps/Src/ReadTritium.c

```diff
@@ -101,16 +104,39 @@ void Task_ReadTritium(void *p_arg){
 					Car_Velocity = ((Car_Velocity / 160934) * 10); //Converting from m/h to mph, multiplying by 10 to make value "larger" for displaying
 
 					UpdateDisplay_SetVelocity(Car_Velocity);
+				}
+
+				case MC_BUS:{
+					memcpy(&Bus_Voltage, &dataBuf.data[0], sizeof(float));
+					memcpy(&Bus_Current, &dataBuf.data[4], sizeof(float));
+
+					//Bus voltage is in bytes 0-4
+					Bus_Voltage = *((float*)(&dataBuf.data[0]));
+
+					//Bus Current is in bytes 4-8
+					Bus_Current = *((float*)(&dataBuf.data[4]));
+
```

Comment:
Why are you doing both a memcpy and a pointer dereference to copy the values? Are these not redundant? 

Imo, just use the memcpy for code clarity.